src/org/joml/Vector2d.java

```diff
@@ -42,11 +42,27 @@ of this software and associated documentation files (the "Software"), to deal
  *
  * @author RGreenlees
  * @author Kai Burjack
+ * @author F. Neurath
  */
 public class Vector2d implements Externalizable, Vector2dc {
 
     private static final long serialVersionUID = 1L;
 
+    /**
+     * Gets the length of a 2-dimensional double-precision vector.
+     * Addresses <a href="https://github.com/JOML-CI/JOML/issues/131">Issue #131</a>
+     *
+     * @param x The vectors x component
+     * @param y The vectors y component
+     *
+     * @return the length of the given vector
+     *
+     * @author F. Neurath
+     */
+    public static double vecLength2d(double x, double y) {
```

Comment:
Please rename to simply "length()". There is obviously no need to have the "vec" prefix and the "2d" suffix, since we are already in that class.

Reply:
Also, please move the method to the length() instance methods.